### DISCUSSION TITLE
Suppress an alert

### DISCUSSION THREAD

**Discussion by cerberus10748** [Security-Onion-Solutions/securityonion#6252] [General] [answered]
Hello, 
I am looking to figure out how to suppress logon success messages from showing on my alerts. These are coming from my DC and Wazuh i believe.

**Comment by dougburks** [comment, marked as answer]
https://docs.securityonion.net/en/2.3/wazuh.html#tuning-rules

**Reply by cerberus10748** [reply]
That is perfect! Thanks Doug, I Completely missed that section in the guide when I looked.
